prow/git/v2/adapter.go

```diff
@@ -2,11 +2,21 @@ package git
 
 import (
 	"errors"
+	"fmt"
+	"strings"
 
 	"k8s.io/test-infra/prow/git"
 	"k8s.io/test-infra/prow/github"
 )
 
+func OrgRepo(full string) (string, string, error) {
+	if strings.Count(full, "/") != 1 {
+		return "", "", fmt.Errorf("full repo name %s does not follow the org/repo format", full)
+	}
+	parts := strings.Split(full, "/")
```

Comment:
Nit: Just adding a lencheck here would achieve the same as the strings.Count. But TIL there is a strings.Count.